### ISSUE TITLE
WP-image to allow preload prop

### ISSUE THREAD

**Issue by drewbaker** [funkhaus/fuxt#32]
Would be great to have a `preload` prop on wp-image that does this: https://web.dev/preload-responsive-images/

Should be false by default.

**Comment by izzygld** [comment]
perhaps we should look at [nuxt-image](https://image.nuxtjs.org/) to take car of the speed/sizing and breakpoints.

**Comment by drewbaker** [comment]
Yeah I've seen that module, it's an interesting idea. I'd need to investigate how best to use that with WordPress serving the images. Probably just give it the full file URL and let the `nuxt-img` component handle everything else.

`wp-image` does a lot of other things too (captions, videos) so I'd have to figure out how to play well with that too.
